en/identity-server/next/docs/guides/organization-management/cleanup-organization-resources.md

```diff
@@ -0,0 +1,10 @@
+# Resources Cleanup for Deleted Organizations
```

Comment:
Let's point this doc user organization deletion section in manage-organizations.md